packages/client/src/profile/graphql/index.ts

```diff
@@ -30,3 +30,121 @@ export const getProfileByAddress = (address: Hash) =>
         }
       }
       `;
+
+export const getProfiles = (numberPerPage?: number, offset?: number, searchQuery?: string) => {
+  const pagination = numberPerPage ? 'first: ' + numberPerPage + ', skip: ' + offset : '';
+  let condition = ', where: {';
+  condition += searchQuery ? `, handle_contains_nocase: "${searchQuery}"` : '';
+  condition += '}';
+
+  return `
+              {
+                users(orderBy: rating, orderDirection: desc ${pagination} ${condition}) {
+                  id
+                  address
+                  handle
+                  userStats {
+                    numReceivedReviews
+                  }
+                  rating
+                }
+              }
+              `;
+};
+
+export const getProfileById = (id: string) => `
+      {
+          user(id: "${id}") {
+          id
```

Comment:
check globally, you got some prettier issue

Reply:
prettier does not work on strings, as we store these queries in backticked strings, prettier doesn't affect them
I have manually went through the files to format these (I have shifted to far rleft).